s3torchbenchmarking/src/s3torchbenchmarking/dcp/README.md

```diff
@@ -0,0 +1,134 @@
+## PyTorch's Distributed Checkpoint (DCP) benchmarks
+
+The `dcp` Python package holds all the logic to execute benchmarks for [PyTorch's Distributed Checkpointing][DCP]
+feature against the `s3torchconnector` library.
+
+### Purpose
+
+The benchmarks are designed to test the "save" and "load" mechanisms of PyTorch
+(`torch.distributed.checkpoint[save,load]`), and compare the s3torchconnector library performance vs. other libraries,
+like fsspec.
```

Comment:
lets not mention fsspec directly, but better leave "other libraries and local storage"